ShawMedallion/src/main/java/org/usfirst/frc2450/ShawMedallion/OI.java

```diff
@@ -117,8 +116,8 @@ public OI() {
         SmartDashboard.putData("ElevatorMid", new ElevatorMid());
         SmartDashboard.putData("ElevatorHigh", new ElevatorHigh());
         SmartDashboard.putData("RunElevator", new RunElevator());
-       SmartDashboard.putData("ElevatorToPosition33", new ElevatorToPosition(33));
-       SmartDashboard.putData("ElevatorToPosition10", new ElevatorToPosition(10));
+        SmartDashboard.putData("ElevatorToPosition33", new ElevatorToPosition(33));
+        SmartDashboard.putData("ElevatorToPosition10", new ElevatorToPosition(10));
```

Comment:
Changing "Elevator to Middle" would be more informative on the dashboard and make it more flexible if middle position is changed to 35.